internal/daemon/daemon.go

```diff
@@ -453,8 +456,13 @@ func (d *Daemon) Start(ctx context.Context, started chan struct{}) error {
 	if d.Host == "" {
 		listenAddress := ln.Addr().(*net.TCPAddr)
 		d.System.SetHostname(internal.NormalizeAddress(listenAddress))
+		if d.WebhookHost == "" {
+			d.System.SetWebhookHostname(internal.NormalizeAddress(listenAddress))
+		}
 	}
+
 	d.V(0).Info("set system hostname", "hostname", d.System.Hostname())
+	d.V(0).Info("set webhook hostname", "webhookHostname", d.System.WebhookHostname())
```

Comment:
Nitpick: I would refer not to `"webhookHostname`" but just to `"hostname"`, as the fact it is the hostname for the webhook is made clear in the log message itself, `"set webhook hostname"`.

Reply:
I did it like that, so it is easier to parse out of the logs if somebody wanted to